server/router/router.go

```diff
@@ -233,16 +234,22 @@ func New(opts ...Config) http.Handler {
 		api.POST("/purge", userCookie, rt.purgeEvents)
 
 		api.GET("/login", accountAuth, rt.getLogin)
-		api.POST("/login", rt.postLogin)
-		api.POST("/logout", rt.postLogout)
-
-		api.POST("/change-password", accountAuth, rt.postChangePassword)
-		api.POST("/change-email", accountAuth, rt.postChangeEmail)
-		api.POST("/forgot-password", rt.postForgotPassword)
-		api.POST("/reset-password", rt.postResetPassword)
-		api.POST("/share-account/:accountID", accountAuth, rt.postShareAccount)
-		api.POST("/share-account", accountAuth, rt.postShareAccount)
-		api.POST("/join", rt.postJoin)
+		if rt.oidc == nil {
+			api.POST("/login", rt.postLogin)
+			api.POST("/logout", rt.postLogout)
+
+			api.POST("/change-password", accountAuth, rt.postChangePassword)
+			api.POST("/change-email", accountAuth, rt.postChangeEmail)
+			api.POST("/forgot-password", rt.postForgotPassword)
+			api.POST("/reset-password", rt.postResetPassword)
+			api.POST("/share-account/:accountID", accountAuth, rt.postShareAccount)
+			api.POST("/share-account", accountAuth, rt.postShareAccount)
+			api.POST("/join", rt.postJoin)
+		} else {
+			api.POST("/login", rt.oauthLogin)
```

Comment:
Would there be any harm in always exposing these routes in addition to the existing ones and just 40x  out when someone calls them and OIDC is not configured? I would assume the URLs could be namespaced like `/oidc/login` or something.

Reply:
Sure, namespacing them would be easy. My main thought was: if the operator has an issuer set up, they likely won't use the internal login anyway, so there's no need for the form to be shown. One of my thoughts regarding the UI part was that it might be the easiest option to just use the "Login as operator" button as the "use SSO" button, which is why it's right now colliding.

Reply:
Hmm, then maybe let's defer the decision here until we know what the UI will look like. I'll talk about this with @hendr-ik beginning of next year and we'll feedback on how to proceed here.